components/raftstore/src/errors.rs

```diff
@@ -287,6 +287,14 @@ impl From<Error> for errorpb::Error {
                 e.set_store_peer_id(store_peer_id);
                 errorpb.set_mismatch_peer_id(e);
             }
+            Error::DeadlineExceeded => {
```

Comment:
It is been used, otherwise, we can remove this error type. 
raftstore also does deadline check and then generate `Error::DeadlineExceeded` error.

Reply:
https://github.com/tikv/tikv/blob/213299221806959c95d05d0f2d7368e597fa9281/components/raftstore/src/store/fsm/peer.rs#L656

Reply:
I see, a `deadline checker` is introduced in the `tikv_util` and it's used many places including raftstore and read pool.

Reply:
Yes, the `DeadlineError` error will convert to `Error::DeadlineExceeded` in the next line.

Reply:
How about moving the conversion logic from L291 to L297 to the `deadline.rs` in the `tikv_util` component. The deadline error is actually converted from the deadline error there, we could convert it back and into the error type `errorpb::Error` there.
This could avoid the duplication.

Reply:
great,done.